index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>
+        JS Documentation
+      </header>
+      <!--This area of the code shows the navigation of the page-->
+      <ul>
+        <li>
+          <a href="#Introduction" class="nav-link">Introduction</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know" >What you should already know</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#JavaScript_and_Java" >JavaScript and Java</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#Hello_world" >Hello world</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#Variables" >Variables</a>
+        </li>
+      </ul>
+    </nav>
+
+
+   <main id="main-doc">
+
+
+    <section class="main-section" id="Introduction">
+      <header>
+        Introduction
+      </header>
+      <p>
+        JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them.
+
+      </p>
+
+      <p>
+        JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
+
+      </p>
+
+      <UL>
+        <li>
+          Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM)
+        </li>
+
+        <li>
+          Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server.
+        </li>
+    </UL>
+
+
+
+    </section>
+    <section class="main-section" id="What_you_should_know">
+      <header>
+        What you should already know
+      </header>
+      <p>
+        This guide assumes you have the following basic background:
+
+      </p>
+
+      <p>
+        A general understanding of the Internet and the World Wide Web (WWW).
+
+      </p>
+
+      <ul>
+        <li>
+          A general understanding of the Internet and the World Wide Web (WWW).
+        </li>
+        <li>
+          Good working knowledge of HyperText Markup Language (HTML).
+
+        </li>
+
+        <li>
+          Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.
+        </li>
+      </ul>
+
+      
+    </section>
```

Comment:
Thanks for trying to organize your code. That's a very good step. But just an issue. Please you can try reducing the spaces(gaps) after each line of code. It would look more interesting to follow along.